taskcat/_cfn/template.py

```diff
@@ -161,6 +172,9 @@ def _find_children(self) -> None:  # noqa: C901
                 if child_name:
                     # for child_url in child_name:
                     children.add(child_name)
+        self._process_children(children)
+
+    def _process_children(self, children: set) -> None:
```

Comment:
Appease `pylint` :P (Too many branches in function.)

Alternative naming could be, `_add_children` or `_find_children2`. (Because naming is hard.)